pkgs/development/python-modules/nxt-python/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, fetchgit
+, isPy3k
+, pyusb
+, pybluez
+, pytest
+, git
+}:
+
+buildPythonPackage rec {
+  version = "3.0.1";
+  pname = "nxt-python";
+  format = "setuptools";
+
+  src = fetchgit {
+    url = "https://github.com/schodet/nxt-python.git";
+    rev = version;
+    sha256 = "004c0dr6767bjiddvp0pchcx05falhjzj33rkk03rrl0ha2nhxvz";
+    branchName= "master";
```

Comment:
```suggestion
```
This is not needed with the rev being set

Reply:
Ah I saw it used in other places, will remove.